build.gradle

```diff
@@ -134,7 +134,7 @@ configurations.all {
     if (it.state != Configuration.State.UNRESOLVED) return
     resolutionStrategy {
         force "joda-time:joda-time:${versions.joda}"
-        force "com.fasterxml.jackson.core:jackson-core:2.13.4"
+        force "com.fasterxml.jackson.core:jackson-core:2.14.0"
```

Comment:
should we maybe just inherit jackson from core?

Reply:
will check if it is feasible in the main branch PR and merge this branch PR.